src/VirtoCommerce.Platform.Core/PlatformOptions.cs

```diff
@@ -58,5 +58,15 @@ public class PlatformOptions
         /// Extensions of the files that can be uploaded to the server by the platform
         /// </summary>
         public string[] FileExtensionsWhiteList { get; set; } = new string[0];
+
+        /// <summary>
+        /// Extend Rest API reference schemas (using the allOf construct) so that contextual metadata.
```

Comment:
```suggestion
        /// Extend Rest API reference schemas (using the allOf construct) so that contextual metadata can be applied to all parameter and property schemas.
```